src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -88,8 +88,8 @@
 public final class ValidateVariants extends VariantWalker {
```

Comment:
This tool is in a different category than the others: VariantEvaluationProgramGroup.java

Reply:
I get an error again when I try to compile it in my machine (./gradlew shadowJar), both with VariantEvaluationProgramGroup.java and VariantEvaluationProgramGroup.class.

Error:
wm07e-9a6:gatk mkanaszn$ ./gradlew shadowJar
:compileJava
/Users/mkanaszn/Broad_Institute/Code/gatk_ssh/gatk/src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java:97: error: cannot find symbol
        programGroup =  VariantEvaluationProgramGroup.java
                        ^
  symbol: variable VariantEvaluationProgramGroup
1 error
:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.495 secs

Reply:
It works now, problem was that I didn't import the class.